uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java

```diff
@@ -421,7 +421,7 @@ private void queueDeleteEvent(WatchEvent object,
                                                       final IndexerDispatcher dispatcher) throws DisposedException {
                             final Path oldPath = context.getOldPath();
                             // ignore delete events for dot files, because dot files are not indexed
-                            if (!oldPath.getFileName().toString().startsWith(".")) {
+                            if (!isDotFile(oldPath)) {
```

Comment:
`dispatcher.offer(new DeletedFileEvent(oldPath))` int this if-branch is not covered in test

Reply:
I have added a unit test for it as well